### PR TITLE
Add unit tests for tex2Wiki

### PR DESCRIPTION
**Pull request by pwang00**

### CODE REVIEW

**Comment by pwang00** [comment]
@physikerwelt should we merge?

**Comment by physikerwelt** [comment]
no, do not merge. The tests are failing https://travis-ci.org/DRMF/DRMF-Seeding-Project/builds/136558065

File: Azeem/test/test_getEq.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from unittest import TestCase
+from tex2Wiki import getEq
+from tex2Wiki import setup_label_links
+class TestGetEq(TestCase):
+
+    def test_getEq(self):
+        self.assertEqual('', getEq(''))
+        self.assertEqual('', getEq(''))
```
Comment (physikerwelt) [inline Azeem/test/test_getEq.py]: you do not need to run the same test multiple times

Comment (pwang00) [inline Azeem/test/test_getEq.py]: Yeah those were placeholders.  I'm updating those right now.

File: Azeem/test/test_unmodLabel.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from unittest import TestCase
+from tex2Wiki import unmodLabel
+from tex2Wiki import setup_label_links
+class TestModLabel(TestCase):
+
+    def test_unmodLabel(self):
+        self.assertEqual('Formula:DLMF:25.2:E1', unmodLabel('Formula:DLMF:25.2:E1'))
+        self.assertEqual('Formula:DLMF:15.2:E1', unmodLabel('Formula:DLMF:15.02:E1'))
```
Comment (physikerwelt) [inline Azeem/test/test_unmodLabel.py]: From the test I get the impression that the funciton unmodLabel does nothing. Try to add better test cases, that it get's clear what the function does.

**Comment by physikerwelt** [comment]
maybe you can discuss which tests make most sense with @HowardCohl ?

**Comment by physikerwelt** [comment]
Did you discuss about test cases with @HowardCohl ?

**Comment by HowardCohl** [comment]
@physikerwelt @pwang00 

I did talk with Philip, but honestly my main concern is with functionality. I have no idea what tests should be applied until we have some functionality. Do we have functionality? Has this been tested? Otherwise what stage are we at? What functions are there? What do they do? Do they work? Is this the right strategy?

**Comment by physikerwelt** [comment]
@HowardCohl the test should test individual functions of the program.
See
https://github.com/DRMF/DRMF-Seeding-Project/pull/52/files
 for the current tests.
Does that cover the essence of the functions?
I'll merge this pull request now, since those tests are certainly better than nothing.

**Pull request merged**